tests/discovery_test.js

```diff
@@ -4,13 +4,341 @@ import chai, { expect } from "chai";
 import chaiAsPromised from "chai-as-promised";
 import sinon from "sinon";
 
+import {getUserIDHash, registerUserURL, retrieveUserURL} from "../lib/index.js";
+import { fakeServerResponse } from "./test_utils.js";
+
 chai.use(chaiAsPromised);
 chai.should();
 chai.config.includeStack = true;
 
-describe("getUserIDHash", () => {
-  it("should return the same hash for the same userid", () => {
-    const hash1 = getUserIDHash('shweta');
-    expect(hash1).to.eql(getUserIDHash('shweta'));
+const root = typeof window === "object" ? window : global;
+var localStorage;
+var LocalStorage = require('node-localstorage').LocalStorage;
+localStorage = new LocalStorage('./lib/localStorage');
```

Comment:
you can probably remove line 15 and write:

    var localStorage = new LocalStorage('./lib/localStorage');